### PR TITLE
Bump gcc version for mingw-w64

### PR DESCRIPTION
**Pull request by michael-loki**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by Moisan** [comment]
@BrewTestBot test this please

File: Formula/mingw-w64.rb [hunk 1]
```diff
@@ -26,9 +26,9 @@ class MingwW64 < Formula
   end
 
   resource "gcc" do
-    url "https://ftp.gnu.org/gnu/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
-    mirror "https://ftpmirror.gnu.org/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
-    sha256 "64baadfe6cc0f4947a84cb12d7f0dfaf45bb58b7e92461639596c21e02d97d2c"
+    url "https://ftp.gnu.org/gnu/gcc/gcc-9.1.0/gcc-9.1.0.tar.gz"
```
Comment (Moisan) [inline Formula/mingw-w64.rb]: Does `mingw-w64` depends on this particular version of `gcc`? If not we should remove this resource and use `depends_on "gcc"` instead, unless `gcc` should be built differently for `mingw-w64`.

Comment (michael-loki) [reply Formula/mingw-w64.rb]: It is built differently for `mingw-w64` (but I agree it would be nice to just use the "upstream" version of gcc).

**Review by Moisan — CHANGES_REQUESTED** [review]
The revision would also need to be bumped. Simply change `revision 1` to `revision 2`. This build trigger a rebuild of the bottle.

**Comment by Moisan** [comment]
Thank you @michael-loki  for your first contribution! Without people like you, we couldn't run this project.

**Pull request closed without merging**